### PR TITLE
Markdown 382

### PR DESCRIPTION
**Pull request by JeffAbrahamson**
This doesn't complete many issues, but it does seem to work and advances the state of affairs.

### CODE REVIEW

File: transport_nantes/topicblog/tn_links.py [hunk 1]
```diff
@@ -60,12 +60,14 @@
...
-def render_inclusion_tag_to_html(tag_source, tag_name):
+def render_inclusion_tag_to_html(tag_source, tag_name, kwargs):
     """Render an inclusion tag to a string.
     """
 
     template_string = f"{{% load {tag_source} %}}{{% {tag_name} %}}"
-    template_context = {'csrf_token': settings.csrf_token}
+    template_context = {'csrf_token':  settings.csrf_token,
+                        'mailinglist': kwargs['mailinglist']}
+    print('tc', template_context)
```
Comment (Shriukan33) [inline transport_nantes/topicblog/tn_links.py:70]: Dev print here

File: transport_nantes/topicblog/tn_links.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
     [[don:adhésion]]((text))   equivalent to {% bouton_join "text" %}
     [[don:fixed|1]]((text))    equivalent to {% fixed_amount_donation_button 1 "text" %}
 
-    [[news:name]]((text))      equivalent to {% show_mailing_list %}
+    [[news:name]]((text))      equivalent to {% show_mailing_list name %}
```
Comment (Shriukan33) [inline transport_nantes/topicblog/tn_links.py:41]: The news MD tag is pretty confusing in the name, I'd expect news to be somewhat related to articles (The news ..) not to show a subscribe to mailing list block.

This isn't new for the name, it's just that it made me realize it wasn't intuitive for me.

File: transport_nantes/topicblog/test_tn_links.py [hunk 3]
```diff
@@ -51,10 +51,14 @@ def test_buttons(self):
         self.assertEqual(self.parser.transform('[[don:fixed|5]]((give!))'), \
                          don.fixed_amount_donation_button(5, 'give!'))
 
-    def test_news(self):
-        settings.csrf_token = 'some text that must exist'
-        self.assertEqual(self.parser.transform('[[news:kangaroo]]((aardvark))'), \
-                         render_inclusion_tag_to_html('newsletter', 'show_mailing_list'))
+    ## Disabled due to captcha matching.
```
Comment (Shriukan33) [inline transport_nantes/topicblog/test_tn_links.py:54]: Faced the captcha validation problem with the login form, turns out there is a way to get away with it in tests. 
 Can see here how it's done : 
 https://github.com/transport-nantes/tn_web/commit/e077ef0173a990cbfc76d25367972e682043913b#diff-e4c08575b99cb05e4808251bfb547bc1a429237cd6af2ac65d313cf1e59c02b0

TLDR; Captcha has a [test mode](https://django-simple-captcha.readthedocs.io/en/latest/advanced.html#captcha-test-mode)

File: transport_nantes/topicblog/tn_links.py (hunk 2 above)
Comment (JeffAbrahamson) [reply transport_nantes/topicblog/tn_links.py:41]: Yes, I agree.  I want to change the name, but that seems best as a separate change, properly staged (introduce new name, deploy, make sure old is out of service, then retire old).

File: transport_nantes/topicblog/test_tn_links.py (hunk 3 above)
Comment (JeffAbrahamson) [reply transport_nantes/topicblog/test_tn_links.py:54]: Yeah, I looked at that.  But it makes the captcha accept whatever it gets so that the submit validates, it doesn't make it produce the same string.  And here I was testing that the two functions generate the same html.

File: transport_nantes/topicblog/tn_links.py (hunk 2 above)
Comment (Shriukan33) [reply transport_nantes/topicblog/tn_links.py:41]: Agreed

File: transport_nantes/topicblog/test_tn_links.py (hunk 3 above)
Comment (Shriukan33) [reply transport_nantes/topicblog/test_tn_links.py:54]: Ha, got it. Indeed, seems like we shall have to do without it.

**Comment by JeffAbrahamson** [comment]
Back to you.

**Comment by Shriukan33** [comment]
LGTM
